pkg/rook/logs.go

```diff
@@ -0,0 +1,16 @@
+package rook
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+)
+
+func SetCephLogLevel(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, storageClusterNamespace string, component, subsystem, level string) {
+
```

Comment:
How about a log statement that indicates the setting was successfully applied? And if it fails, it already prints the error?

Reply:
added logging statement

Reply:
``` ❯ bin/odf set ceph-log-level osd.0 crush 1/10
Info: successfully set the log levels for "osd.0" subsystem "crush" as "1/10"
```